_maps/_basemap.dm

```diff
@@ -1,7 +1,7 @@
 //#define LOWMEMORYMODE //uncomment this to load centcom and runtime station and thats it.
 
 #include "map_files\generic\CentCom.dmm"
-
+#define FORCE_MAP "_maps/multiz_debug.json"
```

Comment:
Reminder to remove this when you are done

Reply:
See: TODO list